reactive/kubernetes_master.py

```diff
@@ -1458,6 +1458,13 @@ def build_kubeconfig(server):
         kubeconfig_path = os.path.join(os.sep, 'home', 'ubuntu', 'config')
         # Create the kubeconfig on this system so users can access the cluster.
 
+        with open(kubeclientconfig_path, 'r') as conf:
```

Comment:
You meant `kubeconfig_path` vs `kubeclientconfig_path` here, right?

Also, I think on initial deployment, `/home/ubuntu/config` won't exist yet.  That'll lead to a `FileNotFoundError` on this `open`.  Consider a `try/pass` or check with something like `isfile()` to workaround that.